pages/calender.jsx

```diff
@@ -1,114 +1,62 @@
 import { Link } from "react-router-dom";
+import { useState, useRef, useEffect } from "react";
+import TaskList from "../components/calendar_components/TaskList";
+import { v4 as uuid } from "uuid";
 
 const ToDoList = () => {
+  const [tasks, setTasks] = useState([]);
+  const taskNameRef = useRef();
+  const LOCAL_STORAGE_KEY = "taskApp.tasks";
```

Comment:
Would move that to line 5. You don't need that every time created when the component is re-rendered.